building/windows.md

```diff
@@ -0,0 +1,173 @@
+# Windows
+
+It is possible to build Phoenix-rtos images for all targets and run `ia32-generic-qemu` on Windows using Cygwin or WSL.
+
+## Cygwin
+
+In the beginning, we need to download and set the QEMU environment.
+
+To do that we need to simply go to QEMU official website and select mirror which is targeted to our system.
+
+* [QEMU](https://www.qemu.org/download/#windows)
+
+![OS bit selection](_images/qemu-os-select.png)
+
+Next up we need to download Cygwin from the official website. It is a collection of Open Source tools that provide
+functionality similar to a Linux distribution on Windows.
+
+* [Cygwin](https://cygwin.com/install.html)
+
+<details>
+<summary>Cygwin installation/update process (click to expand)</summary>
+&nbsp;
+
+* The installation process is in this case also a package manager. Cygwin will ask of type of installation in our case
+it will be from internet
+
+![cygwin-download-source](_images/cygwin-download-source.png)
+
+* After that install process will ask about placement of root and local package directory
+
+![cygwin-root-dir](_images/cygwin-root-dir.png)
+![cygwin-local-directory](_images/cygwin-local-directory.png)
+
+* Cygwin have capability of using host internet connection, but it must be configured with our preferences (In most

```

Comment:
```suggestion
* Cygwin has capability of using host's internet connection, but it needs to be configured with our preferences (In most

```
I don't like my version of the sentence after "but", maybe you have a better idea